partner_firstname/README.rst

```diff
@@ -47,7 +47,7 @@ supply the *last name* and *first name* (just once per contact).
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/134/10.0
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
```

Comment:
Change the readme also, from:

> You can configure some common name patterns for the inverse function
in Settings > Configuration > General settings

To:
> You can configure some common name patterns for the inverse function
in Settings > General settings